.github/workflows/pre-commit.yml

```diff
@@ -11,4 +11,7 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - uses: actions/setup-python@v4
+    - name: Skip hook when run for 'push'
+      if: ${{ github.event_name == 'push' }}
+      run: echo "SKIP=no-commit-to-branch" >> $GITHUB_ENV
```

Comment:
Why not add it always? This pre-commit makes sense only on the user-side anyway

Reply:
Right you are, it does not make sense to run it on GitHub: 205e6db3